packages/discord.js/src/structures/NewsChannel.js

```diff
@@ -13,7 +13,7 @@ class NewsChannel extends BaseGuildTextChannel {
    * Adds the target to this channel's followers.
    * @param {TextChannelResolvable} channel The channel where the webhook should be created
    * @param {string} [reason] Reason for creating the webhook
-   * @returns {Promise<NewsChannel>}
+   * @returns {Promise<Snowflake>} Returns created target webhook id.
```

Comment:
```suggestion
   * @returns {Promise<Snowflake>} Returns the created webhook id in the target channel.
```

Reply:
Seems like one might as well return the whole object instead of just the id?

Reply:
discord.js doesn't cache them. Doesn't mean the user can do it themselves.

Reply:
I don't see any way to do so, unless fetching the webhook, which is not ideal. Returning either the object or snowflake will force to check if it's one or not. If you have any code suggestions share them

Reply:
This is irrelevant to the point, and another point to raise is this is not a future-proof solution. If Discord decides to make a sparkling new feature related to this and more properties get returned, it would be a breaking change to change the return type. This should be an object.

Reply:
I just realised what you mean